### PR TITLE
#19646 Log error nicely

### PR DESCRIPTION
**Pull request by freddyucv**
Throw Exception when a file can not be add into the BundleOutput

### CODE REVIEW

**Review by victoralfaro-dotcms — APPROVED** [review]
LGTM

File: dotCMS/src/main/java/com/dotcms/publishing/output/AddFileNotPossibleException.java [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+        super(cause);
+        this.filePath = filePath;
+    }
+
+    @Override
+    public String getMessage(){
+        final Throwable cause = this.getCause();
+
+        final String message = cause.getMessage().contains("Not a directory") ?
+                "At least one subfolder is really a file" : cause.getMessage();
```
Comment (dsilvam) [inline dotCMS/src/main/java/com/dotcms/publishing/output/AddFileNotPossibleException.java:20]: "At least one subfolder already exists as a file"

File: dotCMS/src/main/java/com/dotcms/publishing/output/AddFileNotPossibleException.java [hunk 2]
```diff
@@ -0,0 +1,25 @@
+package com.dotcms.publishing.output;
+
+
+import java.io.IOException;
+
+public class AddFileNotPossibleException extends IOException {
```
Comment (dsilvam) [inline dotCMS/src/main/java/com/dotcms/publishing/output/AddFileNotPossibleException.java:6]: FileCreationException

File: dotCMS/src/main/java/com/dotcms/publishing/output/AddFileNotPossibleException.java (hunk 1 above)
Comment (freddyucv) [reply dotCMS/src/main/java/com/dotcms/publishing/output/AddFileNotPossibleException.java]: done https://github.com/dotCMS/core/pull/20419/commits/7d032810fb0eae3135ca03c5637a4965f5570828#diff-0ebd5a060797bda7f7fb9b64f984234f4796b2e1e9288179ccc443f278c04682R20

**Review by dsilvam — APPROVED** [review]

**Pull request merged**